lib/shelley/test/data/cardano-node-shelley/genesis.yaml

```diff
@@ -6,7 +6,7 @@ protocolParams:
   protocolVersion:
     minor: 0
     major: 0
-  decentralisationParam: 0.97 # means 3% decentralised
+  decentralisationParam: 0.1 # means 90% decentralised
```

Comment:
This was very high to avoid too frequent rollbacks until we have a proper transaction scheduler. I don't think you'll get anything merged with 90% decentralization :)

Reply:
We can try, but I am afraid this is going to block many PRs after that :/